examples/quick/main.qml

```diff
@@ -7,14 +7,16 @@ import QtQuick.Window 6.5
 import QtLocation 6.5
 import QtPositioning 6.5
 
+import QtLocation.MapLibre 3.0
+
 Window {
     id: window
     width: Qt.platform.os === "android" ? Screen.width : 512
     height: Qt.platform.os === "android" ? Screen.height : 512
     visible: true
 
     property bool fullWindow: false  // toggle full map with the 'F' key
-    property var coordinate: QtPositioning.coordinate(59.91, 10.75)  // Oslo
+    property var coordinate: QtPositioning.coordinate(41.874, -75.789)

```

Comment:
What is the new place? :)

Reply:
Some undetermined place from the US East Coast so the example weather radar overlay can be used. I think the comment is not necessary.